backend/src/database.cpp

```
@@ -74,8 +95,8 @@ namespace database
             {
                 const char *error_msg = sqlite3_errmsg(db_);
                 fprintf(stderr, "SQLite error: %s\n", error_msg);
```

**question**: Do we want this to throw an exception if below it aborts the code?

---

Sure!

---

Also, what other places need better error checking that throws errors instead of just fprintf errors?